tests/h/views/admin_groups_test.py

```diff
@@ -139,13 +140,20 @@ def test_raises_not_found_if_unknown_group(self, pyramid_request):
         with pytest.raises(HTTPNotFound):
             GroupEditController(pyramid_request)
 
-    def test_read_renders_form(self, pyramid_request, group):
+    def test_read_renders_form(self, pyramid_request, factories, group):
         pyramid_request.matchdict = {'pubid': group.pubid}
+        factories.Annotation(groupid=group.pubid)
+        factories.Annotation(groupid=group.pubid)
+
         ctrl = GroupEditController(pyramid_request)
 
         ctx = ctrl.read()
 
         assert ctx['form'] == self._expected_form(group)
+        assert ctx['pubid'] == group.pubid
+        assert ctx['group_name'] == group.name
+        assert ctx['num_members'] == len(group.members)
+        assert ctx['num_annotations'] == 2
```

Comment:
From a quick skim through other views it seems like we tend to use `{thing}_count` for template vars like this.